src/index.js

```diff
@@ -10,7 +10,7 @@ export default function fileLoader(content) {
 
   const query = loaderUtils.getOptions(this) || {};
   const configKey = query.config || 'fileLoader';
-  const options = this.options[configKey] || {};
+  const options = this.options && this.options[configKey] || {};
```

Comment:
I'm not sure what I would write a test for, without setting up and running throw webpack. The loader assumes this.options is provided by webpack unconditionally its not. It only exists in v2+ if the user provides the LoaderOptionsPlugin, a plugin that exists only to help ease migration from vq. This is just simple mistake on the webpack api. Either this or add to the Readme saying the LoaderOptionsPlugin is required